### DISCUSSION TITLE
It appears your account may be based in a U.S.-sanctioned region

### DISCUSSION THREAD

**Discussion by YanaGlad** [community/community#30280] [Other Feature Feedback, Questions, & Ideas]
Hello! Github says "It appears your account may be based in a U.S.-sanctioned region" and this is why I can't create private repositories or login to GitHub via JetBrains products.  
It happened 3 years ago when I entered Crimea for 2 weeks. It's been 3 years, I haven't been in Crimea since that time but problem still stays with me... And it doesn't matter where I am now. It stays with me even if I go to the United States. None of people around me are facing same problem

**Comment by airtower-luna** [comment]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!
